pkg/upload/js.go

```diff
@@ -247,13 +247,13 @@ func AddSources(sourceMapContents map[string]interface{}, sourceMapPath string,
 		if sources, ok := sections.([]map[string]interface{}); ok {
```

Comment:
Project root can be removed from `AddSources` too

Reply:
Ahh yes, this has been removed now.